### ISSUE TITLE
droid -Nr errors out when encounting a broken symlink

### ISSUE THREAD

**Issue by mistydemeo** [digital-preservation/droid#39]
If droid runs on a symlink whose target doesn't exist,  a `-Nr` scan of a directory will exit immediately and print a backtrace for the exception.

For example, my home directory has a .zlogin symlink left over from when I used zsh; when scanning my home directory, this was the output:

```
16:42:59,783 ERROR [main] NoProfileRunCommand:174 - error processing files
java.io.FileNotFoundException: ./.zlogin (No such file or directory)
    at java.io.FileInputStream.open(Native Method)
    at java.io.FileInputStream.<init>(FileInputStream.java:140)
    at uk.gov.nationalarchives.droid.command.action.NoProfileRunCommand.execute(NoProfileRunCommand.java:167)
    at uk.gov.nationalarchives.droid.command.DroidCommandLine.run(DroidCommandLine.java:137)
    at uk.gov.nationalarchives.droid.command.DroidCommandLine.processExecution(DroidCommandLine.java:193)
    at uk.gov.nationalarchives.droid.command.DroidCommandLine.main(DroidCommandLine.java:168)
java.io.FileNotFoundException: ./.zlogin (No such file or directory)
```

It should probably swallow exceptions and continue on, printing a message to stderr.

**Comment by paulyoung84** [comment]
This issue has been added to the DROID backlog, it will be assigned an appropriate priority but is unlikely to be included in the next release.

**Comment by sparkhi** [comment]
Fixed in 6.6.0
